### PR TITLE
Add simple combo docs

### PR DESCRIPTION
**Pull request by wnvko**
Add docs for new `IgxSimpleCombo` component. Additionally update related docs in `IgxCombo` component.
Closes # 

### Checklist:
 
 - [x] check topic's TOC/menu and paragraph headings
 - [ ] link to other topics using `../relative/path.md`
 - [x] at the References section at the end of the topic add links to topics, samples, etc
 - [x] reference API documentation instead of adding a section with API

<br />

 - [x] use valid component names - [Data] Grid, `IgxSelectComponent`, `<igx-combo>`
 - [x] use spell checker tool (VS Code, Grammarly, Microsoft Editor)
 - [x] add inline `code blocks` for the names of classes / tags / properties
 - [x] add language descriptor for the ```code blocks```
 - [x] check broken links (use browser add-on)
 - [x] check if sample is working and fully visible in the topic
 - [ ] check if sample is working and fully visible in the StackBlitz
 - [ ] check if code blocks match the code in StackBlitz demo

<br />

 - [x] follow SEO guidelines to fill topic's metadata ([SEO guidelines](https://infragisticsinc297.sharepoint.com/:w:/g/Groups/marketing/EWz9InT4FDlErHCumxsKGY4Bd8H03yhRWxDFk47luRz-_Q?e=S5wWcx))

<br />

 - [ ] do not resolve requested changes (leave that to the reviewer)
 - [ ] add `pending-localization` label when the review of the PR is done
 - [ ] add a member from the localization team to translate it

### CODE REVIEW

File: en/components/simple-combo.md [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+---
+
+# Single Select ComboBox
+
+The Angular Simple ComboBox component is a modification of [ComboBox component](combo.md) that allows single selection. Simple combobox is an editable input that allows users to choose an option from a predefined list of items. The Ignite UI for Angular Simple ComboBox Component provides also filtering capabilities, selections of items, grouping and adding custom values to a dropdown list. It can be used as an alternative to the HTML select tag and has several out-of-the-box features such as data binding (local and remote), filtering, grouping, custom templates, custom values, and more.
+
+
+## Angular Simple ComboBox Example
+
+In this Angular Simple ComboBox example, you can see how users can select the chart's trend line type. In addition, the Simple ComboBox expose keyboard navigation and custom styling capabilities.
```
Comment (jackofdiamond5) [inline en/components/simple-combo.md:14]:
```suggestion
In this Angular Simple ComboBox example, you can see how users can select the chart's trend line type. In addition, the Simple ComboBox exposes keyboard navigation and custom styling capabilities.
```

File: en/components/simple-combo.md [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+    imports: [
+        ...
+        IgxSimpleComboModule,
+        ...
+    ]
+})
+export class AppModule {}
+```
+
+Then, in the template, you should bind the [igx-simple-combo]({environment:angularApiUrl}/classes/igxsimplecombocomponent.html) with some data.
```
Comment (jackofdiamond5) [inline en/components/simple-combo.md:52]:
```suggestion
Then, in the template, you should bind the [igx-simple-combo]({environment:angularApiUrl}/classes/igxsimplecombocomponent.html) to some data.
```

File: en/components/simple-combo.md [hunk 3]
```diff
@@ -0,0 +1,366 @@
...
+
+Since the simple combobox is bound to an array of complex data (i.e. objects), we need to specify a property that the control will use to handle the selected items. The control exposes two `@Input` properties - [valueKey]({environment:angularApiUrl}/classes/igxsimplecombocomponent.html#valuekey) and [displayKey]({environment:angularApiUrl}/classes/igxsimplecombocomponent.html#displaykey):
+
+ - `valueKey` - *Optional, recommended for object arrays* - Specifies which property of the data entries will be stored for the simple combobox's selection. If `valueKey` is omitted, the simple combobox value will use references to the data entries (i.e. the selection will be an array of entries from `igxSimpleCombo.data`).
+ - `displayKey` - *Required for object arrays* - Specifies which property will be used for the items' text. If no value is specified for `displayKey`, the simple combobox will use the specified `valueKey` (if any).
+
+In our case, we want the simple combobox to display the `name` of each city and the simple combobox value to store the `id` of each city. Therefore, we are providing these properties to the simple combobox's `displayKey` and `valueKey`, respectively:
```
Comment (jackofdiamond5) [inline en/components/simple-combo.md:77]:
```suggestion
In our case, we want the simple combobox to display the `name` of each city and its value to store the `id` of each city. Therefore, we are binding these properties as values to the simple combobox's `displayKey` and `valueKey`, respectively:
```

**Review by Lipata — APPROVED** [review]

File: en/components/simple-combo.md [hunk 4]
```diff
@@ -0,0 +1,366 @@
...
+```html
+<igx-simple-combo [data]="cities" [displayKey]="'name'" [valueKey]="'id'"></igx-simple-combo>
+```
+
+> [!Note]
+> When the data source is comprised of a simple type (e.g. `string[]`, `number[]`), **do not** specify a `valueKey` and `displayKey`.
+
+### Two-Way Binding
+
+The simple combobox component fully supports two-way data-binding with `[(ngModel)]` as well as usage in [template driven](https://angular.io/guide/forms) and [reactive](https://angular.io/guide/reactive-forms) forms. The simple combobox selection can be accessed either through two-way binding or through the [selection API](#selection-api). We can pass an item of the same type as the ones in the simple combobox's selection (based on `valueKey`) and any time one changes, the other is updated accordingly.
```
Comment (jackofdiamond5) [inline en/components/simple-combo.md:88]:
```suggestion
The simple combobox component fully supports two-way data-binding with `[(ngModel)]` as well as usage in [template driven](https://angular.io/guide/forms) and [reactive](https://angular.io/guide/reactive-forms) forms. The simple combobox selection can be accessed either through two-way binding or through the [selection API](#selection-api). We can pass in an item of the same type as the ones in the simple combobox's selection (based on `valueKey`) and any time one changes, the other is updated accordingly.
```

Comment (jackofdiamond5) [reply en/components/simple-combo.md:88]: Also, the last sentence is kind of difficult to read, can we rephrase it?

File: en/components/simple-combo.md [hunk 5]
```diff
@@ -0,0 +1,366 @@
...
+    public selectedCity: { name: string, id: string } = this.cities[0];
+}
+```
+
+
+### Selection API
+
+The simple combobox component exposes API that allows getting and manipulating the current selection state of the control.
+
+One way to get the simple combobox's selection is via the [selection]({environment:angularApiUrl}/classes/igxsimplecombocomponent.html#selection) property. It returns a value which correspond to the selected item, depending on the specified [valueKey](#data-value-and-display-properties) (if any).
```
Comment (jackofdiamond5) [inline en/components/simple-combo.md:126]:
```suggestion
One way to get its selection is via the [selection]({environment:angularApiUrl}/classes/igxsimplecombocomponent.html#selection) property. It returns a value which corresponds to the selected item, depending on the specified [valueKey](#data-value-and-display-properties) (if any).
```

File: en/components/simple-combo.md [hunk 6]
```diff
@@ -0,0 +1,366 @@
...
+In our example, `selection` will return the selected city's `id`:
+
+```typescript
+export class MySimpleComboComponent {
+    ...
+    public selection: string = this.simpleCombo.selection;
+}
+```
+
+Using the selection API, you can also change the simple combobox's selected item without user interaction with the control - via a button click, as a response to an Observable changing, etc. For example, we can implement a button that selects a city, using the [select()]({environment:angularApiUrl}/classes/igxsimplecombocomponent.html#select) method:
```
Comment (jackofdiamond5) [inline en/components/simple-combo.md:137]:
```suggestion
Using the selection API, you can also change the simple combobox's selected item without user interaction with the control - via a button click, as a response to an Observable changing, etc. For example, we can implement a button that selects a city, using the [select]({environment:angularApiUrl}/classes/igxsimplecombocomponent.html#select) method:
```

File: en/components/simple-combo.md [hunk 7]
```diff
@@ -0,0 +1,366 @@
...
+
+```html
+<igx-simple-combo [data]="cities" [displayKey]="'name'" [valueKey]="'id'"></igx-simple-combo>
+<button igxButton (click)="selectFavorite()">Select Favorite</button>
+```
+
+When clicking the button, the London city will be added to the simple combobox's selection:
```
Comment (jackofdiamond5) [inline en/components/simple-combo.md:144]:
```suggestion
When the button is clicked, `London` will be added to the simple combobox's selection:
```

File: en/components/simple-combo.md [hunk 8]
```diff
@@ -0,0 +1,366 @@
...
+    @ViewChild(IgxSimpleComboComponent, { read: IgxSimpleComboComponent, static: true })
+    public simpleCombo: IgxSimpleComboComponent;
+    ...
+    selectFavorites(): void {
+        this.simpleCombo.select('UK01');
+    }
+}
+```
+
+The simple combobox also fires an event every time its selection changes - [selectionChanging()]({environment:angularApiUrl}/classes/igxsimplecombocomponent.html#selectionchanging). The emitted event arguments, [ISimpleComboSelectionChangingEventArgs]({environment:angularApiUrl}/interfaces/isimplecomboselectionchangingeventargs.html), contain information about the selection prior to the change, the current selection and the displayed item. The event can also be cancelled, preventing the selection update with the new  item.
```
Comment (jackofdiamond5) [inline en/components/simple-combo.md:157]:
```suggestion
The simple combobox also fires an event every time its selection changes - [selectionChanging]({environment:angularApiUrl}/classes/igxsimplecombocomponent.html#selectionchanging). The emitted event arguments, [ISimpleComboSelectionChangingEventArgs]({environment:angularApiUrl}/interfaces/isimplecomboselectionchangingeventargs.html), contain information about the selection prior to the change, the current selection and the displayed item. The event can also be cancelled, preventing the selection from taking place.
```

File: en/components/simple-combo.md [hunk 9]
```diff
@@ -0,0 +1,366 @@
...
+<igx-simple-combo [data]="cities" [displayKey]="'name'" [valueKey]="'id'"
+           (selectionChanging)="handleCityChange($event)">
+</igx-simple-combo>
+```
+
+<div class="divider--half"></div>
+
+## Keyboard Navigation
+
+When simple combobox is closed and focused:
```
Comment (jackofdiamond5) [inline en/components/simple-combo.md:171]:
```suggestion
When the simple combobox is closed and focused:
```

File: en/components/simple-combo.md [hunk 10]
```diff
@@ -0,0 +1,366 @@
...
+- `ArrowDown` or `Alt` + `ArrowDown` will open the simple combobox's drop down.
```
Comment (jackofdiamond5) [inline en/components/simple-combo.md:172]:
```suggestion
- `ArrowDown` or `Alt` + `ArrowDown` will open the simple combobox's dropdown.
```

File: en/components/simple-combo.md [hunk 11]
```diff
@@ -0,0 +1,366 @@
...
+
+> [!NOTE]
+> Any other key stroke will be handled by the input.
+
+When simple combobox is opened and list item is focused:
```
Comment (jackofdiamond5) [inline en/components/simple-combo.md:177]:
```suggestion
When the simple combobox is opened and an item in the list is focused:
```

File: en/components/simple-combo.md [hunk 12]
```diff
@@ -0,0 +1,366 @@
...
+- `ArrowDown` will move to the next list item. If the active item is the last one in the list and custom values are enabled, the focus will be moved to the Add item button.
+
+- `ArrowUp` will move to the previous list item. If the active item is the first one in the list will close the list.
```
Comment (jackofdiamond5) [inline en/components/simple-combo.md:180]:
```suggestion
- `ArrowUp` will move to the previous list item. If the active item is the first one in the list, the list will close.
```

File: en/components/simple-combo.md [hunk 13]
```diff
@@ -0,0 +1,366 @@
...
+
+- `Home` will move to the first list item.
+
+- `Space` will select/deselect the active list item.
+
+- `Enter` will select/deselect the active list item and will close the list.
+
+- `Esc` will close the list.
+
+When combobox is opened, allow custom values are enabled and add item button is focused:
```
Comment (jackofdiamond5) [inline en/components/simple-combo.md:192]:
```suggestion
When the simple combobox is opened and allow custom values are enabled, and add item button is focused:
```

File: en/components/simple-combo.md [hunk 14]
```diff
@@ -0,0 +1,366 @@
...
+
+<code-view style="height: 540px;"
+           data-demos-base-url="{environment:demosBaseUrl}"
+           iframe-src="{environment:demosBaseUrl}/lists/simple-combo-cascading" alt="Angular Cascading Scenario Example">
+</code-view>
+
+<div class="divider--half"></div>
+
+### Template Configuration
+The API of simle combobox is used to get the selected item from one component and load data source for the next one, as well clear the selection and data source when needed.
```
Comment (jackofdiamond5) [inline en/components/simple-combo.md:211]:
```suggestion
The API of simple combobox is used to get the selected item from one component and load data source for the next one, as well clear the selection and data source when needed.
```

File: en/components/simple-combo.md [hunk 15]
```diff
@@ -0,0 +1,366 @@
...
+### Component Definition
+```typescript
+export class SimpleComboCascadingComponent implements core.OnInit {
+    public selectedCountry: Country;
+    public selectedRegion: Region;
+    public selectedCity: City;
+    public countriesData: Country[];
+    public regionData: Region[] = [];
+    public citiesData: City[] = [];
+    ngOnInit(): void {
```
Comment (jackofdiamond5) [inline en/components/simple-combo.md:244]:
```suggestion
    public ngOnInit(): void {
```

**Review by jackofdiamond5 — APPROVED** [review]

**Review by Lipata — APPROVED** [review]

**Pull request merged**